netlify/functions/repositories/UserRepository.ts

```diff
@@ -89,6 +90,44 @@ class UserRepository {
       .where("user_id", "=", userId)
       .execute();
   }
+
+  async createClubInvite(clubId: string) {
```

Comment:
I don't think this belongs in the user repository. Maybe move to club repsitory?